resources/com.sap.piper/pipeline/cloudSdkStageDefaults.yml

```diff
@@ -111,6 +111,9 @@ stages:
       artifactDeployment:
         configKeys:
           - 'nexus'
+      nexusUpload:
```

Comment:
This does not have any effect, since the stage artifactDeployment does not read the runStep condition. I would suggest to either align the artifact deployment stage or remove the runStep condition.

Reply:
I thought it would at least have the effect to enable the stage if the step is configured. If the stage runs, it executes nexusUpload unconditionally.

Reply:
Yes you are right. This is actually required in case someone configures the step instead of the stage, since then it is not activated with the workaround I introduced above in lines 111-113.